app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -495,5 +497,13 @@ def high_profile_groups_links
     def roles_links
       item.roles.distinct.pluck(:content_id)
     end
+
+    def default_news_image
+      return unless item.default_news_image
+      {
+        url: item.default_news_image.file_url,
```

Comment:
Tracing back through whitehall code I think we'll need to set a size on this too:

https://github.com/alphagov/whitehall/blob/1311c133ee97c51e06c62a67139c68454222876d/app/presenters/lead_image_presenter_helper.rb#L35-L42

Reply:
Ah thanks, I think I missed the bit of logic that checked if the content type was SVG.